### PR TITLE
Add AddressSanitizer support to fibers.

### PR DESCRIPTION
**Pull request by JohanEngelen**

### CODE REVIEW

**Comment by JohanEngelen** [comment]
https://github.com/ldc-developers/druntime/pull/152

**Comment by JohanEngelen** [comment]
Manually tested on macOS and Ubuntu now. I'm satisfied with it now.

Depends on https://github.com/ldc-developers/ldc/pull/2984

File: tests/lit.site.cfg.in [hunk 1]
```diff
@@ -27,6 +27,7 @@ config.gnu_make_bin        = "@GNU_MAKE_BIN@"
 config.ldc_host_arch       = "@LDC_HOST_ARCH@"
 config.ldc_with_lld        = "@LDC_WITH_LLD@" == "ON"
 config.spirv_enabled       = @LLVM_SPIRV_FOUND@ +0 # LLVM_SPIRV_FOUND is blank if not set
+config.rt_supports_sanitizers = "@RT_SUPPORT_SANITIZERS@" == "ON"
```
Comment (kinke) [inline tests/lit.site.cfg.in]: I'm not sure, but a cached `RT_SUPPORT_SANITIZERS` CMake variable may be `1` instead of `ON`.

Comment (kinke) [reply tests/lit.site.cfg.in]: Going to merge anyway in order to move forward.

**Pull request merged**